javascriptv3/example_code/bedrock-runtime/config/foundation_models.js

```diff
@@ -50,18 +50,6 @@ export const FoundationModels = Object.freeze({
     module: () => import("../models/ai21_labs_jurassic2/jurassic2.js"),
     invoker: (/** @type {Module} */ module) => module.invokeModel,
   },
-  LLAMA2_CHAT_13B: {
```

Comment:
Why break with this convention?

Reply:
Because the examples I've added are all top-level quickstarts without any boilerplate so customers can easily copy/paste. More advanced examples will follow, but currently customers need simple examples to know how to even get started. Last time I checked, there was literally not a single non-Python example available anywhere on the internet.